Rakefile

```diff
@@ -20,6 +20,44 @@ require 'bundler/gem_tasks'
 
 require "rspec/core/rake_task"
 
+require "active_record"
+
+root = File.expand_path(__dir__)
+ActiveRecord::Tasks::DatabaseTasks.env = ENV["ENV"] || "development"
+ActiveRecord::Tasks::DatabaseTasks.database_configuration = YAML.load(File.read(File.join(root, 'config/database.yml')))
```

Comment:
Prefer `YAML.load_file(filename)`

Reply:
Yeah already changed to that in ff225839d851921ec9e0585cb9ce3995e3c5fabe